.gitignore

```diff
@@ -3,6 +3,7 @@
 *.jl.mem
 docs/build/
 docs/site/
+docs/src/examples/*.md
```

Comment:
why ignore these?

Reply:
these are generated by literate right? the source literate file and markdown files in the same folder might be redundant
if you want, we can keep them in the docs folder - that way apart from https://jump.dev/DiffOpt.jl/dev, one can access docs here too